BasicRdl/ViewModels/CategoryBrowserViewModel.cs

```diff
@@ -152,6 +152,16 @@ public override void ComputePermission()
         public override void PopulateContextMenu()
         {
             base.PopulateContextMenu();
+
+            if (this.SelectedThing == null || this.SelectedThing.ContainedRows.Count == 0)
```

Comment:
Can this not be abstracted to a higher base class? to avoid repeating this everywhere?

Reply:
Agreed. Change applied